MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -658,7 +659,9 @@ - (void)processLogBufferAfterCrash {
         if (serializedLog && serializedLog.length && serializedLog.length > 0) {
           id<MSLog> item = [NSKeyedUnarchiver unarchiveObjectWithData:serializedLog];
           if (item) {
-            [self.logManager processLog:item withPriority:MSPriority(priority)];
+            
+            // Buffered logs are used sending their own channel. It will never contain more than 20 logs
+            [self.logManager processLog:item withPriority:(MSPriority)priority andGroupID:@"CrashBuffer"];
```

Comment:
I doubt this is a reason of why you are keeping priority even after introducing groupID.